### PR TITLE
Bluetooh section updated

### PR DESCRIPTION
**Pull request by jcarolinares**
## What This PR Changes
- Bluetooh section updated to match the operation manual. The section is now more practical and coherent.

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

File: content/hardware/06.nicla/boards/nicla-voice/tutorials/user-manual/content.md [hunk 1]
```diff
@@ -1325,122 +1325,129 @@ Serial1.println("Hello world!");
...
+- A Bluetooth® Low Energy connection is constantly verified in the `loop()` function, being the build-in LED in red while looking for a connection. When a central device connects to the Nicla Voice, its built-in LED will change its color to blue. The code then enters into a loop that constantly reads the voltage level from an analog input and maps it to a percentage value between 0 and 100. The voltage level is printed to the Serial Monitor and transmitted to the central device over the defined Bluetooth® Low Energy characteristic.
+
+You can use the [nRF Connect for Mobile](https://www.nordicsemi.com/Products/Development-tools/nrf-connect-for-mobile) app from Nordic Semiconductor® to test the functionality of the example code shown below. nRF Connect is a powerful tool that allows you to scan and explore Bluetooth Low Energy® devices and communicate with them.
```
Comment (jhansson-ard) [inline content/hardware/06.nicla/boards/nicla-voice/tutorials/user-manual/content.md:1439]:
```suggestion
You can use the [nRF Connect for Mobile](https://www.nordicsemi.com/Products/Development-tools/nrf-connect-for-mobile) app from Nordic Semiconductor to test the functionality of the example code shown below. nRF Connect is a powerful tool that allows you to scan and explore Bluetooth® Low Energy devices and communicate with them.

```

Comment (jhansson-ard) [reply content/hardware/06.nicla/boards/nicla-voice/tutorials/user-manual/content.md:1439]: Just a small comment on the trademark use ⭐

**Review by canchebagur — APPROVED** [review]
LGTM!

**Review by canchebagur — APPROVED** [review]
LGTM!

**Pull request merged**